src/main/java/com/nguyenquyhy/discordbridge/utils/ColorUtil.java

```diff
@@ -65,4 +68,11 @@ public static TextColor getColor(Color color) {
         }
         return result;
     }
+    public static String removeColor(String text){
```

Comment:
Good Point. But That brings up something that should be done separate of this PR, migrating to TextTemplates instead of using Strings.